src/vite.rs

```diff
@@ -57,6 +59,18 @@ impl Default for Development {
 }
 
 impl Development {
+    /// ```rust
```

Comment:
Awesome, thanks for adding documentation!

Maybe add a one line description as well? Something like:

> Adds a path as a prefix to included vite client assets, e.g., `"/app"`.

Or if you have a better way of phrasing that :)